### PR TITLE
add a default conf again to make linking easier

### PR DESCRIPTION
**Pull request by yosifkit**
There will be some docs forthcoming to update the hub description on how to use these changes (https://github.com/docker-library/docs/).

### CODE REVIEW

**Comment by yosifkit** [comment]
Here are the docs: https://github.com/docker-library/docs/pull/289

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	if [ ! -s "$defaultConf" ]; then
+		sentry init "$defaultConf"
+	fi
+	
+	line="execfile('$linksConf')"
+	if ! grep -q "$line" "$defaultConf"; then
+		echo "$line" >> "$defaultConf"
+	fi
+	
+	#sentry upgrade
```
Comment (tianon) [inline docker-entrypoint.sh]: lol

File: docker-links.conf.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                os.getenv('SENTRY_DB_USER')
+                or os.getenv('POSTGRES_ENV_POSTGRES_USER')
+                or 'postgres'
+            ),
+            'PASSWORD': (
+                os.getenv('SENTRY_DB_PASSWORD')
+                or os.getenv('POSTGRES_ENV_POSTGRES_PASSWORD')
+                or ''
+            ),
+            'HOST': postgres,
```
Comment (tianon) [inline docker-links.conf.py]: Shouldn't we change this to just be `'postgres'` so it can survive IP changes?

File: docker-links.conf.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                or os.getenv('MYSQL_ENV_MYSQL_USER')
+                or 'root'
+            ),
+            'PASSWORD': (
+                os.getenv('SENTRY_DB_PASSWORD')
+                or os.getenv('MYSQL_ENV_MYSQL_PASSWORD')
+                or os.getenv('MYSQL_ENV_MYSQL_ROOT_PASSWORD')
+                or ''
+            ),
+            'HOST': mysql,
```
Comment (tianon) [inline docker-links.conf.py]: Shouldn't we change this to just be `'mysql'` so it can survive IP changes?

File: docker-links.conf.py [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+            }
+        }
+
+redis = os.getenv('REDIS_PORT_6379_TCP_ADDR')
+if redis:
+    SENTRY_BUFFER = 'sentry.buffer.redis.RedisBuffer'
+    SENTRY_REDIS_OPTIONS = {
+        'hosts': {
+            0: {
+                'host': redis,
```
Comment (tianon) [inline docker-links.conf.py]: Shouldn't we change this to just be `'redis'` so it can survive IP changes?

File: docker-entrypoint.sh (hunk 1 above)
Comment (tianon) [inline docker-entrypoint.sh]: Maybe `# TODO sentry upgrade ?` instead?

Comment (tianon) [inline docker-entrypoint.sh]: (so there's a better chance of us remembering that we were on the fence of whether we could even bother with this)

**Comment by yosifkit** [comment]
PTAL :pray: :heart:

**Comment by tianon** [comment]
LGTM

**Pull request merged**